core/src/main/java/cz/xtf/core/openshift/PodPortForward.java

```diff
@@ -0,0 +1,55 @@
+package cz.xtf.core.openshift;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.LocalPortForward;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PodPortForward implements AutoCloseable {
+
+	private OpenShift openShift;
+	private String podName;
+
+	private LocalPortForward connection;
+
+	public PodPortForward(OpenShift openShift, Pod pod) {
+		this.openShift = openShift;
+		this.podName = pod.getMetadata().getName();
+	}
+
+	public PodPortForward(OpenShift openShift, String dcName) {
```

Comment:
Could we switch constructors order? I consider it more readable if called constructor (method) is bellow instead above.